core/math/basis.cpp

```diff
@@ -89,13 +89,26 @@ Basis Basis::orthogonalized() const {
 	return c;
 }
 
+// Returns true if the basis vectors are orthogonal (perpendicular), so it has no skew or shear, and can be decomposed into TRS.
```

Comment:
Translation is not a part Basis.
```suggestion
// Returns true if the basis vectors are orthogonal (perpendicular), so it has no skew or shear, and can be decomposed into RS (rotation and scale components).
```

Reply:
Based on this comment, it does seem like "scale" in this context is being used to mean a diagonal matrix, not a diagonalizable matrix as I initially presumed it to mean. I would be very careful with this terminology as it suggests that stretching an image along say, the 45 degree line is not a "scale" transformation, but stretching an image along the 0 degree or 90 degree line is.